tests/contrib/celery/test_task.py

```diff
@@ -1,439 +0,0 @@
-import celery
```

Comment:
This is removed entirely. All these cases are tested under `test_integration.py`.